java/androidpayload/library/src/com/metasploit/meterpreter/AndroidMeterpreter.java

```diff
@@ -33,27 +33,7 @@
 import com.metasploit.meterpreter.android.webcam_start_android;
 import com.metasploit.meterpreter.android.webcam_stop_android;
 import com.metasploit.meterpreter.android.android_wlan_geolocate;
-import com.metasploit.meterpreter.stdapi.Loader;
```

Comment:
ah, works out - we need to stick an explicit getuid import here to prefer the android version from PR #142